### PR TITLE
Update fr-fr.h

### PR DESCRIPTION
**Pull request by lilrawry**
correcting some mistakes & the to do section translated.

### CODE REVIEW

**Comment by goatshriek** [comment]
Thanks for putting these updates in!

The static analysis is failing because it has detected a translation but still sees the `todo translate` comment on line 152. If you remove that comment from the string you've translated, it will be good to go.

Also consider translating the other string at line 119 (`L10N_INVALID_STATE_DURING_UTF8_PARSING`), which is the only other French string missing. If you decide to translate this as well, remember to remove the matching translate todo comment on line 117 as well.

**Comment by lilrawry** [comment]
well received with thanks, i'll do that asap.

File: include/private/config/locale/fr-fr.h [hunk 1]
```diff
@@ -151,7 +150,7 @@
 
 // todo translate
```
Comment (goatshriek) [inline include/private/config/locale/fr-fr.h]: This todo line needs to be removed now that you've added a translation.

**Review by goatshriek — CHANGES_REQUESTED** [review]
Almost there! Just one more todo comment to remove and it will be good to go.

File: include/private/config/locale/fr-fr.h (hunk 1 above)
Comment (lilrawry) [reply include/private/config/locale/fr-fr.h]: all good 👍🏼

**Review by goatshriek — APPROVED** [review]

**Comment by lilrawry** [comment]
Hi again,
need anything else let me know 
keep me updated 
have a lovely day.

**Comment by goatshriek** [comment]
All merged, and thanks again for the contribution!

**Comment by lilrawry** [comment]
My pleasure 🤍✨

**Pull request merged**